internal/storage/fake_storage_util.go

```diff
@@ -45,6 +62,21 @@ func CreateFakeStorageServer(objects []fakestorage.Object) (*fakestorage.Server,
 	})
 }
 
+func CreateFakeStorageServerWithTestObject() (*fakestorage.Server, error) {
```

Comment:
please order the methods in a file appropriately. It will be good to keep interface methods first and then other public methods unless interface methods internally call these public methods.

Reply:
fixed